fix: update Microsoft Defender ATP integration parameters to use encr… (#41204)
demisto/content: Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_20_9.md

#### Integrations

##### Microsoft Defender for Endpoint

- Updated the Docker image to: *demisto/crypto:1.0.0.4578119*.

> - Updated the Microsoft Defender Advanced Threat Protection integration parameters: *ID*, *Token* to an encrypted types

Reviewer: ```suggestion
- Updated the Microsoft Defender Advanced Threat Protection integration parameters: *ID* and *Token* to be encrypted.
```
